Source/EMsoftWrapperLib/CMakeLists.txt

```
@@ -42,6 +87,8 @@ target_include_directories(EMsoftWrapperLib
       ${CLFortran_INSTALL}/include
       $<BUILD_INTERFACE:${EMsoftHDFLib_BINARY_DIR}>
       $<BUILD_INTERFACE:${EMOpenCLLib_BINARY_DIR}>
+      PUBLIC ${PYTHON_INCLUDE_DIR}
+      PUBLIC ${NUMPY_INCLUDE_DIR}
```

```suggestion
      PUBLIC Python3::NumPy
```

With the change in the find_package call, you should be able to link to the target instead.

---

The change leads to an error:
```
 CMake Error in Source/EMsoftWrapperLib/CMakeLists.txt:
   Target "EMsoftWrapperLib" INTERFACE_INCLUDE_DIRECTORIES property contains
   path:

     "/home/zihaod/EMsoftPublic/Source/EMsoftWrapperLib/Python3::Numpy"

   which is prefixed in the source directory.
```
Any idea how to fix it (remove the prefix)? Thanks!

---

fixed. Should still be
```
PUBLIC ${PYTHON_INCLUDE_DIR}
PUBLIC ${NUMPY_INCLUDE_DIR}
```

```
Python3::NumPy
```
is for library linkage.

---

Sorry my mistake, I misread the statement. You should be able to do it with target_link_libraries:
`target_link_libraries(EMsoftWrapperLib PUBLIC Python3::NumPy)`

That should add the appropriate includes directories automatically.